docs/technical-documentation/updating_drupal.md

```diff
@@ -0,0 +1,67 @@
+# Updating Drupal 
+
+## Always create backs ups (DB and files) before updating
+
+**Before updating either Drupal core or Drupal modules:**
+* Back up both your files and database. Having a complete backup makes it easy to revert to the prior version if the update fails.
+* Optionally, if you made manual modifications to files like .htaccess, composer.json, or robots.txt, copy them somewhere easy to find. Because after you've installed the new Drupal core, you will need to re-apply the changes. For example, Acquia Dev Desktop places a .htaccess file in the top-level directory and without it, only the homepage on your site will work.
+
+**Warning:** Always revert to a backup if you get a fatal error in the update process.
+
+## What is Composer
+It is recommended by Drupal.org and the Islandora 8 community to use Composer with Drupal 8 for various tasks. 
+
+"[Composer](https://getcomposer.org/) is a [dependency manager](https://en.wikipedia.org/wiki/Package_manager) for PHP. Drupal core uses Composer to manage core dependencies like Symfony components and Guzzle."
+
+Taken from: https://www.drupal.org/docs/develop/using-composer/using-composer-with-drupal
```

Comment:
Would suggest making this `"[Source](https://www.drupal.org/docs/develop/using-composer/using-composer-with-drupal)"` instead.